tools/klee/main.cpp

```diff
@@ -1330,7 +1333,7 @@ int main(int argc, char **argv, char **envp) {
   // locale and other data and then calls main.
   Function *mainFn = mainModule->getFunction(EntryPoint);
   if (!mainFn) {
-    llvm::errs() << "'" << EntryPoint << "' function not found in module.\n";
+    klee_error("'%s' function not found in module.\n", EntryPoint.c_str());
```

Comment:
Since in this case it is going to terminate (the next line is a return -1), I would rather stick to `klee_error`.